

bool STOP_OPERATIONAL
bool OPERATIONAL_OBSTACLE
